### PR TITLE
Adding jumping to tab of current day

### LINKED ISSUE #51: Make schedule jump to current day

### PR DESCRIPTION
**Pull request by chyku**
Closes #51

### CODE REVIEW

File: app/src/main/java/org/hackillinois/android/view/schedule/DayFragment.java [hunk 1]
```diff
@@ -75,7 +75,7 @@ public void onChanged(List<Event> events) {
     @Override
     public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
-        View view = inflater.inflate(R.layout.fragment_schedule_fragment, container, false);
+        View view = inflater.inflate(R.layout.fragment_schedule_day, container, false);
```
Comment (bstrauch24) [inline app/src/main/java/org/hackillinois/android/view/schedule/DayFragment.java]: Thanks for renaming this 😂

File: app/src/main/java/org/hackillinois/android/view/schedule/ScheduleFragment.java [hunk 2]
```diff
@@ -28,6 +29,18 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         viewPager.addOnPageChangeListener(new TabLayout.TabLayoutOnPageChangeListener(tabLayout));
         tabLayout.addOnTabSelectedListener(new TabLayout.ViewPagerOnTabSelectedListener(viewPager));
 
+        ScheduleViewModel scheduleViewModel = new ScheduleViewModel();
+
+        long time = System.currentTimeMillis();
+
+        if (time < scheduleViewModel.getFRIDAY_END()) {
```
Comment (bstrauch24) [inline app/src/main/java/org/hackillinois/android/view/schedule/ScheduleFragment.java]: Don't mix camelCase with UNDERSCORED_UPPERCASE

Comment (bstrauch24) [reply app/src/main/java/org/hackillinois/android/view/schedule/ScheduleFragment.java]: You could make the FRIDAY_END variable a public static final in the file that it's declared in, or a regular private field with a getter method.

Comment (patrickfeltes) [reply app/src/main/java/org/hackillinois/android/view/schedule/ScheduleFragment.java]: I think this is why Kotlin doesn't want upper case in variable names. It automatically generates getters and setters depending on var/val, so stuff like this happens.

**Pull request merged**